test/views/withdraw/WithdrawSummarySpec.scala

```diff
@@ -0,0 +1,136 @@
+
+  "Withdraw Summary view" when {
+
+    val tstPensionContributionNoPsoDisplaySections = Seq(
+      AmendDisplaySectionModel("PensionsTakenBefore", Seq(
+        AmendDisplayRowModel("YesNo", Some(controllers.routes.AmendsController.amendPensionsTakenBefore("ip2014", "active")), None, "No"))
+      ))
+    val model = AmendDisplayModel("IP2014", amended = true, tstPensionContributionNoPsoDisplaySections, psoAdded = false, Seq(), "£1,100,000")
+    lazy val view = views(model)
+    lazy val doc = Jsoup.parse(view.body)
+
+    s"have a title ${"pla.withdraw.protection.title"}" in {
+      doc.title() shouldBe plaWithdrawTitle
+    }
+
+    "have a navigation tag that" should {
+
+      "have a size of 1" in {
+        doc.select("nav").size() shouldBe 1
+      }
+
+      "have a 'ul' tag with class name of 'breadcrumb-nav__list'" in {
+        doc.select("ul").hasClass("breadcrumb-nav__list") shouldBe true
+      }
+
+      "have a 'ul' tag that has a size of 1" in {
+        doc.select("ul").size() shouldBe 1
+      }
+
+      "have 'li' tag with class name of 'breadcrumb-nav__item'" in {
+        doc.select("li").get(0).hasClass("breadcrumb-nav__item") shouldBe true
+      }
+
+      "have 'a' tag with id of 'account-home-breadcrumb-link'" in {
+        doc.select("a").get(0).id() shouldBe "account-home-breadcrumb-link"
+      }
+
+      s"have text of $plaExistingProtectionsBreadcrumbPTAHome with id of 'account-home-breadcrumb-link'" in {
+        doc.getElementById("account-home-breadcrumb-link").text() shouldBe plaExistingProtectionsBreadcrumbPTAHome
+      }
+
+      "have 'a' tag with id of 'account-home-breadcrumb-link' has a href" in {
+        doc.getElementById("account-home-breadcrumb-link").attr("href") shouldBe FrontendAppConfig.ptaFrontendUrl
+      }
+
+      "have 'a' tag with id of 'existing-protections-breadcrumb-link'" in {
+        doc.select("a").get(1).id() shouldBe "existing-protections-breadcrumb-link"
+      }
+
+      s"have text of $plaExistingProtectionsPageBreadcrumb with id of 'existing-protections-breadcrumb-link'" in {
+        doc.getElementById("existing-protections-breadcrumb-link").text() shouldBe plaExistingProtectionsPageBreadcrumb
+      }
+
+      "have 'a' tag with id of 'existing-protections-breadcrumb-link' has a href" in {
+        doc.getElementById("existing-protections-breadcrumb-link").attr("href") shouldBe controllers.routes.ReadProtectionsController.currentProtections.url
+      }
+
+      "have 'li' tag with class of 'breadcrumb-nav__item' that has text" in {
+        doc.select("li.breadcrumb-nav__item").get(2).text() shouldBe plaExistingProtectionsPageBreadcrumb
+      }
+    }
+
+    "have 'li' tag with class of heading-large that has text" in {
+      doc.select("h1").text() shouldBe plaWithdrawHeadingIP2014
+    }
+
+    "have a table that" should {
+
+      "heading that has a text" in {
+        doc.select("h2").text() shouldBe plaWithdrawSummaryHeading
+      }
+
```

Comment:
Needs a test to see whether the amends section is appearing or not depending on input.